debian/changelog

```diff
@@ -1,3 +1,15 @@
+ceph (17.2.2-1) stable; urgency=medium
+
+  * New upstream release
+
+ -- Ceph Release Team <ceph-maintainers@ceph.com>  Thu, 21 Jul 2022 17:29:31 +0000
+
+ceph (17.2.1-1) stable; urgency=medium
+
+  * New upstream release
+
+ -- Ceph Release Team <ceph-maintainers@ceph.com>  Thu, 23 Jun 2022 14:41:32 +0000
```

Comment:
@djgalloway This PR includes "tools: ceph-objectstore-tool is able to trim pg log dups' entries." and "17.2.1" commits which are already in quincy (merged via https://github.com/ceph/ceph/pull/46827).

Reply:
Yeah, I thought those would disappear out of this PR once https://github.com/ceph/ceph/pull/46827 was merged.  I'm not sure how the commits are broken.

Reply:
I think you need to rebase on top of commit f61704e9a484e828fc16639456e5747f3e54e768.

Reply:
No, scratch that -- I wasn't thinking straight.